### DISCUSSION TITLE
devOps-vancouver-meetup-engineering-empowerment-session-preparations

### DISCUSSION THREAD

**Discussion by giscus** [WorkSafeBC-Common-Engineering/Common.Engineering.Discussion#22] [Technical Blog]
# devOps-vancouver-meetup-engineering-empowerment-session-preparations

https://wsbctechnicalblog.github.io/devOps-vancouver-meetup-engineering-empowerment-session-preparations.html

**Comment by wpschaub** [comment]
Kevin and Martin, how much time do you need for your section(s)? If we divide the 20min by the 5 sections, we roughly have 3-4 minutes per section. I do not need 12 minutes for the first three sections, which would us to focus more on the Nexus and the knowledge sharing?

**Reply by MerlinDarkstar** [reply]
I could do 1 minute per point, 5 points, on Knowledge Enhancement - perhaps a bit more on the first two points.

**Reply by wpschaub** [reply]
Kevin and Martin, perhaps it will be easier to tell me how much time you need for your section (do not cut corners!) and then I can tweak the first sections to use the remaining time. I want to share the 2023 AzDO automation achievements, GH Copilot evaluation results, and mention our big rocks for 2024.
